fluent-plugin-webhdfs.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |gem|
   gem.name          = "fluent-plugin-webhdfs"
-  gem.version       = "0.5.3"
+  gem.version       = "0.6.0rc1"
```

Comment:
If you update code to work with only v0.14, upgrade major version.

http://docs.fluentd.org/articles/plugin-development#plugin-versioning-policy